### DISCUSSION TITLE
Article about logic-analyzer in German magazine Raspberry-Pi-Geek

### DISCUSSION THREAD

**Discussion by bablokb** [gusmanb/logicanalyzer#79] [General]
The article is available in print and here:

https://www.raspberry-pi-geek.de/ausgaben/rpg/2023/09/datenaustausch-zwischen-komponenten-analysieren/

Note that there is currently a paywall, but the magazine usually gives free access to the content after six months. But development is so fast that it already is outdated :-)

**Comment by gusmanb** [comment]
I already have a printed copy on the way 😄 
I have a friend of mine that is living in Berlin so I bought a copy mid August, sent it to his address and he has resent it to me, I hope to receive it very soon!

![imagen](https://github.com/gusmanb/logicanalyzer/assets/4086913/a65412d5-1648-4ef4-b5b7-3c4907e2fff5)


Thanks!
